src/Microsoft.Health.Dicom.SqlServer/Features/Schema/SchemaVersionConstants.cs

```diff
@@ -7,8 +7,8 @@ namespace Microsoft.Health.Dicom.SqlServer.Features.Schema
 {
     public static class SchemaVersionConstants
     {
-        public const int Min = (int)SchemaVersion.V4;
-        public const int Max = (int)SchemaVersion.V9;
+        public const int Min = (int)SchemaVersion.V6;
```

Comment:
We should create a user story to clean up the code to remove these constants and if else logics

Reply:
yup